### PR TITLE
Add gvisor test script for ngo version of occlum

### PR DESCRIPTION
**Pull request by daan3601**

### CODE REVIEW

File: occlum/run_one_tests_on_occlum.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (jessehui) [inline occlum/run_one_tests_on_occlum.sh]: Give some info if the user doesn't input anything.

File: occlum/run_one_tests_on_occlum.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		FILTER=$(cat $BLOCKLIST_DIR/$TEST_CASE | tr '\n' ':')
+		return 0
+	else
+		FILTER=""
+		return 1
+	fi
+}
+
+if [ -e /$TEST_CASE_DIR/$TEST_CASE ];then
+	continue
```
Comment (jessehui) [inline occlum/run_one_tests_on_occlum.sh]: continue shouldn't be used here. Please refine the if statements.

File: occlum/run_one_test_on_occlum.sh [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        else
+                FILTER=""
+        fi
+}
+
+run_testcase(){
+	if [ -e /$TEST_CASE_DIR/$testcase ];then
+		cp $TEST_CASE_DIR/$testcase ./image/bin
+                occlum build
+		if [ -z $subtest ];then #wu zi ce shi
```
Comment (jessehui) [inline occlum/run_one_test_on_occlum.sh]: What is this comment?

**Review by jessehui — COMMENTED** [review]
Looks good overall. Please check the comments.

File: occlum/run_one_tests_on_occlum.sh (hunk 1 above)
Comment (daan3601) [reply occlum/run_one_tests_on_occlum.sh]: Changes have been submitted

File: occlum/run_one_tests_on_occlum.sh (hunk 2 above)
Comment (daan3601) [reply occlum/run_one_tests_on_occlum.sh]: Changes have been submitted

File: occlum/run_one_test_on_occlum.sh (hunk 3 above)
Comment (daan3601) [reply occlum/run_one_test_on_occlum.sh]: has been deleted

**Review by jessehui — APPROVED** [review]
LGTM

**Pull request closed without merging**